submissions/evgeniy24/friends_app/app.js

```diff
@@ -0,0 +1,234 @@
+            return response.json();
+        });
+}
+
+function handleErrors(response) {
+    if (response.status > 400) {
+        throw Error(response.statusText);
+    }
+    return response;
+}
+
+function startApp() {
+    sendRequest('GET', REQUEST_URL)
+    .then( function(data) { 
+        friendsList = data.results;
+        renderFriendsList(friendsList);
+    })
+    .catch( function(err) {
+        renderErrorMessage();
+    });
+}
+
+startApp();
+
+// drawing a list of friends---------------------------
+function renderFriendsList(friends) {
+    FRIENDS_LIST_WRAP.innerHTML = '';
+    
+    friends.forEach(function(friend) {
+        renderFriendsCard(friend);
+    });
+}
+
+// create friends card, add card to the markup-----------
+function renderFriendsCard(friend) {
+    const cardWrap = document.createElement('div');
+    const mainInfo = document.createElement('div');
+    const friendsName = document.createElement('a');
+    const moreWrap = document.createElement('div');
+    const friendPhotoWrap = document.createElement('div')
+    const friendPhoto = document.createElement('img');
+    const friendAge = document.createElement('p');
+    const friendEmail = document.createElement('a');
+    const friendPhone = document.createElement('a');
+    const friendCity = document.createElement('a');
+
+    cardWrap.classList.add('cardWrap');
+    (friend.gender === 'male') ? cardWrap.classList.add('malegradient') : cardWrap.classList.add('femalegradient') 
+    mainInfo.classList.add('mainInfo');
+    moreWrap.classList.add('moreWrap');
+    friendPhotoWrap.classList.add('cardWrap__photo-wrap');
+    friendPhoto.classList.add('cardWrap__photo-img');
+    friendsName.classList.add('mainInfo__friendsName');
+    friendAge.classList.add('mainInfo__age');
+    friendEmail.classList.add('moreWrap__email');
+    friendPhone.classList.add('moreWrap__phone');
+    friendCity.classList.add('mainInfo__city');
+    
+    friendsName.textContent = `${friend.name.first} ${friend.name.last}`;
+    friendAge.textContent = `Age: ${friend.dob.age}`;
+    friendEmail.textContent = `email: ${friend.email}`;
+    friendPhone.textContent = `phone: ${friend.phone}`;
+    friendCity.textContent = `City: ${friend.location.city}`;
+
+    friendPhoto.setAttribute('src', friend.picture.medium);
+    friendsName.setAttribute('href', `https://www.facebook.com/search/top?q=${friend.name.first}%20${friend.name.last}`);
+    friendsName.setAttribute('target', '_blank');
+    friendEmail.setAttribute('href', `mailto:${friend.email}`);
+    friendPhone.setAttribute('href', `tel:>${friend.phone}`);
+    friendCity.setAttribute('href', `https://www.google.com/maps/place/${friend.location.city}$`);
+    friendCity.setAttribute('target', '_blank');
+
+    moreWrap.append(friendPhone, friendEmail);
+    mainInfo.append(friendsName, friendAge, friendCity);
+    friendPhotoWrap.append(friendPhoto);
+    cardWrap.append(friendPhotoWrap, mainInfo, moreWrap);
```

Comment:
Just replace it with template string and `innerHtml`

Reply:
Ok! Done!